bundles/org.openhab.binding.http/src/main/resources/OH-INF/i18n/http_de.properties

```diff
@@ -26,7 +26,7 @@ thing-type.config.http.url.commandMethod.option.PUT = PUT
 thing-type.config.http.url.contentType.label = Inhaltstyp
 thing-type.config.http.url.contentType.description = Der MIME-Content-Typ. Nur für `POST` und `PUT` verwendet.
 thing-type.config.http.url.contentType.option.application/json = anwendung/json
```

Comment:
I guess this is also a wrong translation.
Would there be a possibility to prevent this translatino at all? Like adding  a `notranslate` attribute to the option xml?

Reply:
Just remove it from the default translations properties file. 😉

Reply:
With prevention I meant something that it won’t show up there after generating the file.